src/mmw/js/src/draw/views.js

```diff
@@ -56,21 +56,32 @@ var ToolbarView = Marionette.LayoutView.extend({
     },
 
     onShow: function() {
-        this.selectTypeRegion.show(new SelectAreaView({
-            model: this.model
-        }));
-        this.drawRegion.show(new DrawView({
-            model: this.model
-        }));
-        this.placeMarkerRegion.show(new PlaceMarkerView({
-            model: this.model
-        }));
-        this.resetRegion.show(new ResetDrawView({
-            model: this.model
-        }));
-        this.streamRegion.show(new StreamSliderView({
-            model: this.model
-        }));
+        var draw_tools = settings.get('draw_tools');
+        if (_.indexOf(draw_tools, 'SelectArea') >= 0) {
```

Comment:
I tried to think of a more elegant way of doing this, but instantiating a dynamic class using reflection from a string seemed too indirect and obtuse to be helpful, even though it would reduce some repetition.

If there's a better way of doing this, I'd love to hear about it.